pandas/tests/frame/test_axis_select_reindex.py

```diff
@@ -880,3 +880,47 @@ def test_reindex_multi(self):
         expected = df.reindex([0, 1]).reindex(columns=['a', 'b'])
 
         assert_frame_equal(result, expected)
+
+    def test_align_broadcast_axis(self):
+        # GH 13194
+        # First four tests for DataFrame.align(Index)
+        # For 'right' join
+        df = DataFrame(np.array([[1., 2.], [3., 4.]]), columns=list('AB'))
+        ts = Series([5., 6., 7.])
+
+        result = df.align(ts, join='right', axis=0, broadcast_axis=1)
```

Comment:
try one of these with ``broadcast_axis='columns'`` and one with ``broadcast_axis='index'`` as well